reference/test/stop/iteration.cpp

```diff
@@ -27,6 +27,13 @@ class Iteration : public ::testing::Test {
 };
 
 
+TEST_F(Iteration, NewInterface)
```

Comment:
nit: this test could probably have a more descriptive name, e.g. `SimplifiedInterface` as the other ones, or `FreeFunctionInterface`.